lib/govspeak.rb

```diff
@@ -390,23 +381,25 @@ def self.devolved_options
         lines << %(<div class="govuk-accordion__section ">)
         lines << %(<div class="govuk-accordion__section-header">)
         lines << %(<h2 class="govuk-accordion__section-heading">)
-        lines << %(<div class="govuk-accordion__section-button" id="accordion-default-heading-#{index}">#{heading}</div>)
+        lines << %(<span class="govuk-accordion__section-button" id="accordion-#{@accordion_index}-heading-#{@accordion_heading_index}">#{heading}</span>)
         lines << %(</h2>)
         lines << summary
         lines << %(</div>)
-        lines << %(<div id="accordion-default-content-#{index}" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-#{index}">)
+        lines << %(<div id="accordion-#{@accordion_index}-content-#{@accordion_heading_index}" class="govuk-accordion__section-content" aria-labelledby="accordion-#{@accordion_index}-heading-#{@accordion_heading_index}">)
```

Comment:
I think `accordion_heading_index` should actually be a local variable - so you get
```
accordion-1-content-1
accordion-1-content-2
accordion-1-content-3
accordion-2-content-1
accordion-2-content-2
accordion-2-content-3
```
I think at the moment it is
```
accordion-1-content-1
accordion-1-content-2
accordion-1-content-3
accordion-2-content-4
accordion-2-content-5
accordion-2-content-6
```

Dare I say, a test for 2 accordions might be useful? :)

Reply:
That's a good point! I did the heading index's before the accordion indexes, the heading indexes are now a bit redundant